app/src/store/modules/auth/actions.js

```diff
@@ -103,6 +103,11 @@ export default {
     localStorage.removeItem('displayName')
     localStorage.removeItem('tokenExpiration')
 
+    const meta = router.currentRoute?.meta
+    if (meta?.requiresAuth) {
+      router.push('/nm')
```

Comment:
NIT: Should we show the snack bar here?

Reply:
Should it say 'Logout successful' or something else?

Reply:
oh, or specifically for logging out from auth pages? Maybe I could just call existing `notifyUser` which sends the message `'This page requires login to access'`

Reply:
Sure